src/VirtoCommerce.Platform.Core/Telemetry/SamplingOptions.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.ApplicationInsights.WindowsServer.Channel.Implementation;
+
+namespace VirtoCommerce.Platform.Core.Telemetry
+{
+    /// <summary>
+    /// AppInsights sampling options
+    /// </summary>
+    public class SamplingOptions
```

Comment:
I think better would be to use a single POCO type for all AI settings, and combine everything into one config. Also include **EnableSqlCommandTextInstrumentation** and **IgnoreSqlTelemetryOptions** settings into one options type.